### PR TITLE
Fix #1270b

### PR DESCRIPTION
**Pull request by karysg**
Don't merge yet, only for review.

### CODE REVIEW

File: src-test/org/etools/j1939_84/controllers/part06/Part06Step10ControllerTest.java [hunk 1]
```diff
@@ -180,29 +188,6 @@ public void testFailureForDistanceGreaterThanZero() {
                                         "6.6.10.2.a - Engine #1 (0) reported distance with MIL on > 0");
     }
 
-    @Test
```
Comment (battjt) [inline src-test/org/etools/j1939_84/controllers/part06/Part06Step10ControllerTest.java:183]: Please check why this was removed.

File: src/org/etools/j1939_84/controllers/part04/Part04Step10Controller.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ protected void run() throws Throwable {
             addFailure("6.4.10.2.a - No ECU reported freeze frame data");
         }
 
-        // 6.4.10.2.b. Fail if DTC in freeze frame data does not include the DTC reported in DM12 earlier in this part.
+        // 6.4.10.2.b. Fail if DTC in freeze frame data does not include any DTC reported in DM12 earlier in this part.
```
Comment (battjt) [inline src/org/etools/j1939_84/controllers/part04/Part04Step10Controller.java:84]: Are we sure about this?  If FF includes (1, 2), but DM12 has (1, 2, 3), should a failure be logged?

File: src/org/etools/j1939_84/controllers/part08/Part08Step10Controller.java [hunk 3]
```diff
@@ -75,8 +75,8 @@ protected void run() throws Throwable {
         var packets = filterPackets(dsResults);
         packets.forEach(this::save);
 
-        // 6.8.10.2.a. Fail if DTC(s) reported in the freeze frame does not include either the DTC reported in DM12 or
-        // the DTC reported in DM23 earlier in this part
+        // 6.8.10.2.a. Fail if DTC(s) reported in the freeze frame does not include either any DTC reported in DM12 or
```
Comment (battjt) [inline src/org/etools/j1939_84/controllers/part08/Part08Step10Controller.java:78]: Another place to verify that "any" is interpreted correctly.

**Review by battjt — APPROVED** [review]

File: src-test/org/etools/j1939_84/controllers/part06/Part06Step10ControllerTest.java (hunk 1 above)
Comment (karysg) [reply src-test/org/etools/j1939_84/controllers/part06/Part06Step10ControllerTest.java:183]: Accidental deletion, restored in latest commit.

File: src/org/etools/j1939_84/controllers/part04/Part04Step10Controller.java (hunk 2 above)
Comment (karysg) [reply src/org/etools/j1939_84/controllers/part04/Part04Step10Controller.java:84]: From Eric: "At least one DM12 DTC". Code updated to match in latest commit.

File: src/org/etools/j1939_84/controllers/part08/Part08Step10Controller.java (hunk 3 above)
Comment (karysg) [reply src/org/etools/j1939_84/controllers/part08/Part08Step10Controller.java:78]: Fixed in latest commit.

**Pull request merged**